modules/rhel-compute-requirements.adoc

```diff
@@ -30,11 +30,11 @@ If you have {op-system-base} 7 compute machines that were previously supported i
 For the most recent list of major functionality that has been deprecated or removed within {product-title}, refer to the _Deprecated and removed features_ section of the {product-title} release notes.
 ====
 ** If you deployed {product-title} in FIPS mode, you must enable FIPS on the {op-system-base} machine before you boot it. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening[Installing a RHEL 8 system with FIPS mode enabled] in the {op-system-base} 8 documentation.
++
```

Comment:
@bscott-rh Indenting this properly with the above bullet shifted the below bullets (NetworkManager, etc.) to be second level bullets. That looks like where it was actually supposed to be, but wanted to check with you since this is a visual difference